src/components/TextInputSearch/TextInputSearch.module.scss

```diff
@@ -101,13 +112,9 @@
     width: 80px;
     height: 46px;
   }
-
-  @include g.sm-breakpoint {
-    width: 60px;
-    height: 46px;
-  }
```

Comment:
Removed this as it was causing a bug where the button icon disappeared as soon as the page was resized to this breakpoint